src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -84,6 +43,37 @@ const BatchGroupConfigurationSettings = ({ mutator }) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   []);
 
+  const fetchExportConfig = (id) => {
```

Comment:
still don't understand what the purpose of this var

Reply:
I call this function in `onSave` to update data (exportConfig and credentials)

Reply:
so it should be `useCallback`, as it was before, right?

Reply:
Right